### PR TITLE
testing/zim: new aport

### PR DESCRIPTION
**Pull request by spOOn3113**
http://zim-wiki.org/
A Desktop Wiki Editor

### CODE REVIEW

File: testing/zim/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+pkgver=0.66
+pkgrel=0
+pkgdesc="A Desktop Wiki Editor"
+url="http://zim-wiki.org/"
+arch="all"
+license="GPL2"
+depends="python py-gobject py-xdg py-gtk py-simplejson xdg-utils"
+makedepends=""
+install=""
+#subpackages="$pkgname-dev $pkgname-doc"
```
Comment (Shizmob) [inline testing/zim/APKBUILD]: Could this commented line and all empty variable lines be removed?

File: testing/zim/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Contributor: Adam Nye <adam@spoontech.biz>
+# Maintainer:
```
Comment (Shizmob) [inline testing/zim/APKBUILD]: Not planning on maintaining it? :)

File: testing/zim/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+pkgdesc="A Desktop Wiki Editor"
+url="http://zim-wiki.org/"
+arch="all"
+license="GPL2"
+depends="python py-gobject py-xdg py-gtk py-simplejson xdg-utils"
+makedepends=""
+install=""
+#subpackages="$pkgname-dev $pkgname-doc"
+subpackages="$pkgname-doc"
+source="http://zim-wiki.org/downloads/zim-0.66.tar.gz"
```
Comment (Shizmob) [inline testing/zim/APKBUILD]: Probably more flexible to use $pkgname-$pkgver in the URL here. Makes it more maintainable.

**Comment by Shizmob** [comment]
From the Travis logs, it seems it needs `update-desktop-database` from the `desktop-file-utils` as a build-time dependency. Could you add that package to `makedepends=`?

File: testing/zim/APKBUILD (hunk 3 above)
Comment (spOOn3113) [inline testing/zim/APKBUILD]: All really good points, and I'll go through and make the changes you've requested now.

Its my first attempt at packaging for alpine, so still learning :)

Thanks for the advice.

**Comment by Shizmob** [comment]
Merged, thanks for contributing! :)

**Pull request closed without merging**